apps/total-typescript/src/pages/api/castingwords/create.ts

```diff
@@ -0,0 +1,63 @@
+import {withSentry} from '@sentry/nextjs'
+import {NextApiRequest, NextApiResponse} from 'next'
+import {isValidSignature, SIGNATURE_HEADER_NAME} from '@sanity/webhook'
+
+const secret = process.env.SANITY_WEBHOOK_ORDER_TRANSCRIPT_SECRET
+
+const createCastingwordsOrder = async (
```

Comment:
if this is a sanity webhook receiver it should be named accordingly. somalian to the comment below 

a sanity webhook recover isn't a representative of a single procedure like "order a transcript" it is an indicator that something g has changed on the sanity side and internally will orchestrate all actions that occur in response to the event received.

Reply:
`/api/skill/webhook/sanity`

unless this is different from my suggested workflow and functioning as a Remote Procedure Call